chlorophyll/codeview.py

```diff
@@ -241,5 +246,27 @@ def place_forget(self) -> None:
 
     def destroy(self) -> None:
         for widget in self._frame.winfo_children():
-            tkinter.BaseWidget.destroy(widget)
-        tkinter.BaseWidget.destroy(self._frame)
+            BaseWidget.destroy(widget)
+        BaseWidget.destroy(self._frame)
+
+    def horizontal_scroll(
+        self,
+        first: str | float,
+        last: str | float,
```

Comment:
```suggestion
        last: str | float
```
Maybe remove the comma, and reformat it with black. It would fit on one line